k8s/nginx-ingress/global-configs/stage-nginx-config.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nginx-config
+  namespace: nginx-ingress
+data:
+  proxy-body-size:  "200m"
+  client-max-body-size: "200m"
+  location-snippets: "  
+    auth_request /auth;
+    error_page 500 = @internal_server_error;
+    error_page 404 = @page_not_found;
+    error_page 403 = @access_denied;
+    error_page 401 = @unauthorized;"
+  server-snippets: "
+    location @internal_server_error { return 500 '{\"success\":false, \"message\":\"Something went wrong!\", \"error\":\"Internal Server Error\"}'; }
+    location @page_not_found { return 404 '{\"success\":false, \"message\":\"Resource not found :(\", \"error\":\"Not Found\"}'; }
+    location @access_denied { return 403 '{\"success\":false, \"message\":\"Access denied :|\", \"error\":\"Insufficient Permissions\"}'; }
+    location @unauthorized { return 401 '{\"success\":false, \"message\":\"Invalid authentication\", \"error\":\"Unauthorized.\"}'; }
+
+    location = /auth {      
+      if ($request_method = 'OPTIONS') { return 204; } 
```

Comment:
[Authenticate 'OPTIONS' method for staging](https://github.com/airqo-platform/AirQo-api/pull/1011/commits/7d978b4928ce9b1221d8336ebf5eca901c0189a6)